### DISCUSSION TITLE
Activate QField Cloud PRO

### DISCUSSION THREAD

**Discussion by REDADRISSI** [opengisch/QField#3766] [Q&A]
I am looking to activate the QFIELD CLOUD PRO version. any help ?

Cordially;

**Comment by suricactus** [comment]
What do you mean by "pro" version of QFieldCloud? Can you be more precise what you expect, what you have tried and what you actually get?

**Comment by bsubasi** [comment]
I want to plug on/off with mqtt publish operation

I want you to post example topic plug-s on/off

shellies/shellyplug-s-041471/relay/??????

Ivan Ivanov ***@***.***>, 29 Ara 2022 Per, 13:02 tarihinde
şunu yazdı:
